api/spec/v1.yaml

```diff
@@ -443,6 +461,46 @@ paths:
                 $ref: '#/components/schemas/Account'
         <<: *common-error-responses
 
+  /consensus/accounts/{address}/delegations:
+    get:
+      summary: Returns an account's delegations.
+      parameters:
+        - in: path
+          name: address
+          required: true
+          schema:
+            type: string
+          description: The staking address of the account that delegated.
+          example: *staking_address_1
```

Comment:
This shouldn't need this parameter? It's already in the path?

Reply:
I think we do need to describe the `address`?

Reply:
Oops I misread, I thought this was an additional query param. My mistake.

Reply:
Haha, I actually had one that was redundant earlier in some draft.